content/pt-br/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -0,0 +1,242 @@
+---
+reviewers:
+- floreks
+- maciaszczykm
+- shu-mutou
+- mikedanese
```

Comment:
We can remove this block of `reviewers` for the localized version.

```suggestion
```

Reply:
@dipesh-rawat I removed the reviewers block.